mcp-servers/Dockerfile.markitdown

```diff
@@ -0,0 +1,5 @@
+FROM python:3.13-slim-bullseye
```

Comment:
Just curious if it's possible to use Alpine instead of Debian? For better security.

Reply:
yeah we can.

Reply:
actually probably not, `markitdown-mcp` depends on package `magika`, which requires `onnxruntime` . ONNX Runtime publishes wheels for glibc (manylinux), not for musl (Alpine).

Reply:
actually just realized a new one came out. use trixie